hawkular-wildfly-agent-feature-pack/src/main/resources/configuration/standalone/template.xml

```diff
@@ -18,4 +18,85 @@
 
 -->
 <server xmlns="urn:jboss:domain:4.0">
+
+  <extensions>
+        <?EXTENSIONS?>
+  </extensions>
+
+  <management>
```

Comment:
I don't know what this file is - does this mean the agent feature pack requires these management realms? The agent itself shouldn't need these management realms